subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_aux.c

```diff
@@ -26,6 +26,7 @@
 #include "lll_vendor.h"
 #include "lll_clock.h"
 #include "lll_chan.h"
+#include "lll/lll_df_types.h"
```

Comment:
Why add this?  If required, then use `#include "lll_df_types.h"`

Reply:
fixed